edx_exams/apps/router/views.py

```diff
@@ -0,0 +1,52 @@
+"""
+Views to wrap interactions with the legacy procuring service
```

Comment:
mis-spelled `proctoring`. Also, please reference what the legacy proctoring service referring to: github.com/openedx/edx-proctoring repo.